### ISSUE TITLE
Code Missing: How to Generate task/threshold.txt

### ISSUE THREAD

**Issue by Eren-yeager-zero** [gcd19/D-MoLE#2]
<img width="995" height="178" alt="Image" src="https://github.com/user-attachments/assets/ab6c81d4-aeab-439e-9c36-50380e23de18" />Thank you for your excellent work! During reproduction, I ran into an issue: the program reports that **task/thresholding.txt** is missing (relevant code is shown in the figure within **modeling_internvl_chat.py**. ). After carefully following the provided steps, I did not generate the threshol.txt file, and I couldn’t find any clues in the repository about how it should be created. Any guidance would be greatly appreciated!

**Comment by gcd19** [comment]
The threshold.txt file needs to be created based on your training data characteristics. After training your autoencoder, compute the reconstruction loss on your task's training data and analyze the loss distribution. For example, you can set the threshold slightly higher than the 95th percentile, which enables the routing mechanism to handle natural distribution shifts in test data while filtering out irrelevant tasks. This calibration step ensures optimal performance for your specific dataset. Save the determined threshold values in the threshold.txt file. 

Hope this helps!
